dev-tools/packaging/templates/docker/Dockerfile.tmpl

```diff
@@ -230,6 +230,30 @@ done; \
 (exit $exit_code)
 {{- end }}
 
+# Journald input
+{{- if (eq .BeatName "filebeat") }}
+
+    {{- if (contains .from "wolfi") }}
+    RUN for iter in {1..10}; do \
+            apk update && \
+            apk add --no-interactive --no-progress --no-cache systemd && \
+            exit_code=0 && break || exit_code=$? && echo "apk error: retry $iter in 10s" && sleep 10; \
+        done; \
+        (exit $exit_code)
+    {{- end}}
+
+    {{- if (contains .from "ubi") }}
+    RUN for iter in {1..10}; do \
+            microdnf -y update && \
+            microdnf -y install systemd && \
```

Comment:
What systemd version does this install and how compatible is it with debian? 

I ask because I expect users will frequently use this to read journald logs from debian or ubuntu based k8s nodes so if it isn't compatible we need to clearly document that.

Reply:
UBI at least already exposes us to CVEs in the rest of the OS packages so it doesn't have the same concerns as Wolfi for that problem.

Reply:
> What systemd version does this install and how compatible is it with debian?
> 
> I ask because I expect users will frequently use this to read journald logs from debian or ubuntu based k8s nodes so if it isn't compatible we need to clearly document that.

Is not it sufficiently documented here? https://github.com/elastic/beats/pull/44056/files#diff-7a8b715909f505c1dc033e00d9fb48d50576d9c128e1c50538fe697218b97ff7R11-R21

Reply:
I think almost, that doesn't explain why this is important or what the compatibility rules are. It just says users should check.

What is journactl's version compatibility policy? Backwards compatibility (new versions can read old versions) I assume, but does it also provide forward compatibility (old versions can read new versions of journal files)?

If the Filebeat version is 255 but the host system is 257 what does that mean for users?